daml-lf/language/daml-lf.bzl

```diff
@@ -18,17 +18,29 @@ LF_VERSIONS = [
     "1.7",
     "1.8",
     "1.11",
+    "1.12",
     "dev",
 ]
 
 # The subset of LF versions accepted by the compiler in the syntax
 # expected by the --target option.
-COMPILER_LF_VERSIONS = ["1.dev" if ver == "dev" else ver for ver in LF_VERSIONS]
+COMPILER_LF_VERSIONS = [
+    "1.6",
+    "1.7",
+    "1.8",
+    "1.11",
+    "1.dev",
+]
 
 # We need Any in DAML Script so we require DAML-LF >= 1.7
 SCRIPT_LF_VERSIONS = [ver for ver in COMPILER_LF_VERSIONS if ver != "1.6"]
 
-LF_VERSION_PACKAGE_DIGITALASSET = {"1.6": "digitalasset", "1.7": "digitalasset", "1.8": "digitalasset", "1.11": "daml"}
+LF_VERSION_PACKAGE_DIGITALASSET = {
+    "1.6": "digitalasset",
+    "1.7": "digitalasset",
+    "1.8": "digitalasset",
+    "1.11": "daml",
```

Comment:
Did we include 1.11 here when it was still a preview? If so, we should do the same for 1.12.